tm2/pkg/crypto/keys/client/common.go

```diff
@@ -1,10 +1,6 @@
 package client
 
-import (
-	"fmt"
-	"os"
-	"path/filepath"
-)
+import "github.com/gnolang/gno/gno.land/pkg/config"
```

Comment:
`tm2` should not depend on `gno.land`

Reply:
Moved it to gnovm from now. 1b19dd856ccab4e4160362416d16e44c4c389740

We can't keep this logic in `tm2/pkg/crypto/keys/client/`. because it results in import cycle. 
If you have any better idea let me know.

Reply:
Avoid `tm2` configuration as it powers chains/projects and doesn't need an unpure helper.

`gnovm` requires a user config directory for operations like `gno mod download`. `gno.land` also needs a user config directory for key management and potential local caching. So:

* `tm2`: No UserConfig. Consider a generic helper later.
* `gnovm`: Add helper for equivalent of `$GNOPATH`, used in `gno` CLI operations.
* `gno.land`: Use `gnovm/config.GnoPath()`, create subfolders as needed.